walkthroughs/week-4-libraries/maps/examples/marker/src/main/webapp/script.js

```diff
@@ -13,12 +13,14 @@
 // limitations under the License.
 
 /** Creates a map that shows a single marker. */
+var map;
+var trexMarker;
 function createMap() {
```

Comment:
do u have 2 createMap() functions? Im not sure i see this map on ur website?

Reply:
No, this map is not on my website. I only have one function to create a map in my script.js called initMap(). This file with the createMap() function is one of the many example files given to us in the walkthrough of Week 4 Maps API, but I found that the examples would not run properly because if the map was declared inside the function, script.js said the variable had not been read so I fixed them by moving the declaration right outside the function.